js/filters.js

```diff
@@ -137,7 +137,7 @@
      */
     function UnaryPredicate (column, operator) {
         if (!module.isValidOperator(operator)) {
-            throw new module.InvalidFilterOperatorError("'" + operator + "' is not a valid operator");
+            throw new module.InvalidFilterOperatorError("'" + operator + "' is not a valid operator", this._uri, '');
```

Comment:
This file is not being used by ermrsetjs at all. Just change it back. We should eventually remove this file.

Reply:
Reverting all changes from `js/filters.js`.